ganga/GangaCore/Utility/Monitoring.py

```diff
@@ -0,0 +1,54 @@
+from GangaCore.Utility.Config import getConfig
+
+import os
+from memory_profiler import profile
```

Comment:
I don't think `memory_profiler` is standard. Is the purpose of this branch to go into a release? If so `memory_profiler` needs adding to the requirements.

Reply:
Yes. To install memory_profiler from pip also requires that the python-devel package is installed on any centos/fedora system. If not already in the centos7 docker image, it will need to be added to  ganga/ganga/GangaCore/test/Dockerfile for the testing to work.

Reply:
I added  `memory_profiler` to requirements.txt